### PR TITLE
docs: minor change to user manual

### PR DESCRIPTION
**Pull request by fncolon**
### Pull Request Checklist

_Please make sure to review and check all of these items:_

- [ ] Have you added new tests to prevent regressions?
- [ ] Does `npm run test` or `npm run test-DIALECT` pass with this change (including linting)?
- [x] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [ ] Did you update the typescript typings accordingly (if applicable)?
- [ ] Does the description below contain a link to an existing issue (Closes #[issue]) or a description of the issue you are solving?
- [x] Did you follow the commit message conventions explained in [CONTRIBUTING.md](https://github.com/sequelize/sequelize/blob/main/CONTRIBUTING.md)?

Just a minor changes to increase understanding of the library.

## Todos

- [x] Update virtual explanation
- [x] New example of global / universal hooks ( ref : #13613 )

### CODE REVIEW

**Comment by fncolon** [comment]
Currently I take a look into Hooks and Virtual because it's seems to be interesting to explained more and make it easily indexed by search engine, for example `custom attributes` case in VIRTUAL.

**Comment by fncolon** [comment]
> I marked it as draft since it seems you will update the part on hooks in this PR as well, if I'm incorrect feel free to mark it as 'Ready for review'

Thanks @WikiRik

**Comment by sdepold** [comment]
I wonder if we want to remove getter/setter methods in v7?

**Comment by sdepold** [comment]
> I marked it as draft since it seems you will update the part on hooks in this PR as well, if I'm incorrect feel free to mark it as 'Ready for review'

I have to admit that I'm not a big fan of this. IMHO something is either ready to be reviewed or you need input from the team on how to continue the work (hence make it draft and pro-actively ping the team about a particular aspect). I don't think we should have PRs open for too long in draft state only to add to it later since it's super disturbing at least for my personal PR review flow.

I would rather not open a PR in the first place or create a couple of small PRs. There is nothing wrong with small PRs really.

**Comment by fncolon** [comment]
> I wonder if we want to remove getter/setter methods in v7?

Seems fine, since it's planned a long time ago to be deprecated

**Comment by fncolon** [comment]
> > I marked it as draft since it seems you will update the part on hooks in this PR as well, if I'm incorrect feel free to mark it as 'Ready for review'
> 
> I have to admit that I'm not a big fan of this. IMHO something is either ready to be reviewed or you need input from the team on how to continue the work (hence make it draft and pro-actively ping the team about a particular aspect). I don't think we should have PRs open for too long in draft state only to add to it later since it's super disturbing at least for my personal PR review flow.
> 
> I would rather not open a PR in the first place or create a couple of small PRs. There is nothing wrong with small PRs really.

There's where **todos** section is useful for unfinished PR, draft means the PR is unfinished, that's why github provide the draft feature, but it's ok too if you want to set finish on create PR rules on sequelize, lets get back to the slack consensus for it.

**Comment by sdepold** [comment]
> > I wonder if we want to remove getter/setter methods in v7?
> 
> Seems fine, since it's planned a long time ago to be deprecated

no i mean. deprecation essentially means that it will be removed right? so we should remove it at some point and the next possible step would be v7

**Comment by fncolon** [comment]
> > > I wonder if we want to remove getter/setter methods in v7?
> > 
> > 
> > Seems fine, since it's planned a long time ago to be deprecated
> 
> no i mean. deprecation essentially means that it will be removed right? so we should remove it at some point and the next possible step would be v7

Yep, okay 👍

File: docs/manual/core-concepts/getters-setters-virtuals.md [hunk 1]
```diff
@@ -156,9 +156,9 @@ const user = await User.create({ firstName: 'John', lastName: 'Doe' });
 console.log(user.fullName); // 'John Doe'
 ```
 
-## `getterMethods` and `setterMethods`
+## Deprecated soon: `getterMethods` and `setterMethods`
```
Comment (WikiRik) [inline docs/manual/core-concepts/getters-setters-virtuals.md:159]: Let's just say we deprecate it in v7. Or do we want to keep it for longer?

File: docs/manual/other-topics/hooks.md [hunk 2]
```diff
@@ -116,7 +116,24 @@ You can have many hooks with same name. Calling `.removeHook()` will remove all
 
 ## Global / universal hooks
 
-Global hooks are hooks which are run for all models. They can define behaviours that you want for all your models, and are especially useful for plugins. They can be defined in two ways, which have slightly different semantics:
+Global hooks are hooks which are run for all models. Are especially useful for plugins and they can define behaviours that you want for all your models, for example to allow customization on timestamps using `sequelize.define` on your models:
```
Comment (WikiRik) [inline docs/manual/other-topics/hooks.md:119]:
```suggestion
Global hooks are hooks that are run for all models. They are especially useful for plugins and can define behaviours that you want for all your models, for example to allow customization on timestamps using `sequelize.define` on your models:
```

Just a spelling suggestion

**Review by WikiRik — APPROVED** [review]
Looks good!

File: docs/manual/core-concepts/getters-setters-virtuals.md (hunk 1 above)
Comment (sdepold) [reply docs/manual/core-concepts/getters-setters-virtuals.md:159]: Removal in v7 sounds good. Wanna open a ticket?

Comment (WikiRik) [reply docs/manual/core-concepts/getters-setters-virtuals.md:159]: Woops, I did not read the conversation before mentioning it here. Will make a ticket for it

**Review by sdepold — APPROVED** [review]

File: docs/manual/core-concepts/getters-setters-virtuals.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 Sequelize allows you to define custom getters and setters for the attributes of your models.
 
-Sequelize also allows you to specify the so-called *virtual attributes*, which are attributes on the Sequelize Model that doesn't really exist in the underlying SQL table, but instead are populated automatically by Sequelize. They are very useful for simplifying code, for example.
+Sequelize also allows you to specify the so-called *virtual attributes*, which are attributes on the Sequelize Model that doesn't really exist in the underlying SQL table, but instead are populated automatically by Sequelize. They are very useful to create custom attributes which is also could simplify your code, for example.
```
Comment (dilyanpalauzov) [inline docs/manual/core-concepts/getters-setters-virtuals.md:5]: > which is also could simplify your code

Without *is*.

Comment (fncolon) [reply docs/manual/core-concepts/getters-setters-virtuals.md]: Thank you

**Comment by sdepold** [comment]
https://github.com/sequelize/sequelize/projects/2#card-74642984

File: docs/manual/core-concepts/getters-setters-virtuals.md (hunk 1 above)
Comment (fncolon) [reply docs/manual/core-concepts/getters-setters-virtuals.md]: Changed ✔️

**Pull request merged**